### ISSUE TITLE
int object is not iterable toHex

### ISSUE THREAD

**Issue by ballle98** [earlephilhower/aquaweb#6]
```
pi@raspberrypi:~/git/aquaweb $ ./aquaweb.py
Creating RS485 port...
RS485           : ready
Attempting to auto-detect emulation settings, wait 15 seconds...
...Detected new-style Aqualink PDA.
Detection completed...
Creating PDA emulator...
Creating web server...
Main loop begins...
Started httpserver on port 8080
Traceback (most recent call last):
  File "/home/pi/git/aquaweb/./aquaweb.py", line 883, in <module>
    main()
  File "/home/pi/git/aquaweb/./aquaweb.py", line 877, in main
    screen.processMessage(ret, i)
  File "/home/pi/git/aquaweb/./aquaweb.py", line 627, in processMessage
    print("unk: cmd=" + toHex(ret['cmd']) + " args=" + toHex(ret['args']))
  File "/home/pi/git/aquaweb/./aquaweb.py", line 652, in toHex
    return ''.join(format(x, "02x") for x in blist)
TypeError: 'int' object is not iterable
^CException ignored in: <module 'threading' from '/usr/lib/python3.9/threading.py'>
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 1428, in _shutdown
    lock.acquire()
KeyboardInterrupt:
```

**Comment by ballle98** [comment]
After this fix I see the following

```
pi@raspberrypi:~/git/aquaweb $ ./aquaweb.py
Creating RS485 port...
RS485           : ready
Attempting to auto-detect emulation settings, wait 15 seconds...
...Detected new-style Aqualink PDA.
Detection completed...
Creating PDA emulator...
Creating web server...
Main loop begins...
Started httpserver on port 8080
unk: cmd=1b args=0000
unk: cmd=1b args=0100
^CKeyboard exit requested.
```

**Comment by earlephilhower** [comment]
Fixed by #7
